fed/mosaic-sns/src/main/java/org/eclipse/mosaic/fed/sns/ambassador/TransmissionSimulator.java

```diff
@@ -137,101 +133,115 @@ private boolean isValidSender(String senderName) {
     }
 
     /**
-     * Simulates topolocically-scoped Unicast or Broadcast as direct singlehop transmission.
-     * (NO multi-hops are implemented by now)
-     * <ol>
-     * <li>Verify if configured transmission can be send using the topocast logic
-     * <li>Differentiate between Unicast and Broadcast
-     * <li>Simulate transmission with one hop
-     * </ol>
+     * Simulates topolocically-scoped Unicast (singlehop or multihop transmissions) or Broadcast (only singlehop).
      *
      * @param senderName The Sender of the message.
      * @param dac        {@link DestinationAddressContainer} containing information about the destination for the message.
      * @return a Map containing the summarized transmission results
      */
     protected Map<String, TransmissionResult> sendMessageAsTopocast(String senderName, DestinationAddressContainer dac) {
-        NetworkAddress destinationAddress = dac.getAddress();
-        if (destinationAddress.isAnycast()) { // check for valid destination address
-            log.warn(
-                    "The SNS only supports SingleHopBroadCasts or SingleHopUniCasts when using TopoCasts."
-                            + " The given destination address {} is not valid. No message will be send.",
-                    destinationAddress
-            );
-            return null;
-        }
-        if (dac.getTimeToLive() != SINGLE_HOP_TTL) { // inform about dismissed TTL
-            log.debug("TTL {} will be dismissed and 1 will be used instead. For Topocast, SNS only supports SingleHopBroadCasts.", dac.getTimeToLive());
+        final NetworkAddress destinationAddress = dac.getAddress();
+
+        if (destinationAddress.isBroadcast() && dac.getTimeToLive() != SINGLE_HOP_TTL) {
+            log.debug("SNS only supports single hop broadcasts. TTL {} will be dismissed and 1 will be used instead.", dac.getTimeToLive());
```

Comment:
IMHO this is rather a `warning` if some numbers are overwritten...

for me: where does this overwrite happen?

Reply:
Yes, should be `warn` instead.

When `destinationAddress.isBroadcast()` is true, then `getTimeToLive()` is not used at all, therefore ignored and assumed it's one single hop.